app/models/piece.rb

```diff
@@ -19,4 +19,19 @@ def move_to!(row, col)
     raise "Out of bounds" if row < 0 || row > 7 || col < 0 || col > 7
     update(row: row, col: col)
   end
+
+  def capture_piece(row, col)
+    other_piece = game.piece_at(row, col)
+    return true if game.path_obstructed? && other_piece.user != user
```

Comment:
Yeah, I think `capture_piece` only needs to check if there's a piece there and if it's capturable, since the `path_obstructed?` method will probably be a part of `valid_move?` which should be called before `capture_piece` is.  Or tl;dr `path_obstructed?` should exist soon, but I don't think you need to call it here anyways.

For tests, ping @miljinx for some help, but essentially you'd do something like:

```ruby
describe "capture" do
  it "should capture a piece" do
    game = Game.create
    piece1 = Piece.create(game: game, user_id: 1, row: 0, col: 0)
    piece2 = Piece.create(game: game, user_id: 2, row: 0, col: 1)
    expect(piece1.capture_piece(0, 1)).to eq(true)
    piece2.reload # fetch it from the DB again
    expect(piece2.captured?).to eq(true)
  end

  it "should not capture a player's own piece" do
    game = Game.create
    piece1 = Piece.create(game: game, user_id: 1, row: 0, col: 0)
    piece2 = Piece.create(game: game, user_id: 1, row: 0, col: 1)
    expect(piece1.capture_piece(0, 1)).to eq(false)
    piece2.reload # fetch it from the DB again
    expect(piece2.captured?).to eq(false)
  end

end
```

That's super rough and probably doesn't work, but something like that.  Again, ping Jim and he should be able to assist.